spaces/S000201/README.md

```diff
@@ -3,6 +3,7 @@ uid: S000201
 name: Infinite earring
 aliases:
 - Hawaiian earring
+- Shrinking wedge of circles
```

Comment:
This is a sloppy name, not appropriate here, as this is *not* what is normally understood as a wedge product.

Reply:
I wouldn't use the language "sloppy" myself, but agree that [wedge](https://en.wikipedia.org/wiki/Wedge_sum) can be misleading here and would suggest not including this alias unless it appears in a textbook or widely referenced paper.

Reply:
I absolutely did not mean that @danflapjax was sloppy here. :-) I did mean that whoever author used that name was.

Reply:
I think the pragmatic case for its inclusion is that it appears as "Example 1.25: The Shrinking Wedge of Circles." in Hatcher's [Algebraic Topology](https://pi.math.cornell.edu/~hatcher/AT/AT.pdf), and (I'm pretty sure) this is the most popular introductory algebraic topology book. 

Here are the stats from Google Scholar:
- ["shrinking wedge of circles"](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C9&q=%22shrinking+wedge+of+circles%22&btnG=)
   16 results 
- ["infinite earring"](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C9&q=%22infinite+earring%22&btnG=)
   12 results
- ["hawaiian earring"](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C9&q=%22hawaiian+earring%22&btnG=&oq=%22hawa)
    698 results

I didn't look very closely, but I saw that the textbook [Basic Algebraic Topology and its Applications](https://link.springer.com/book/10.1007/978-81-322-2843-1) refers to it as "Infinite earring or shrinking wedge of circles". 

>  sloppy and misleading name, not appropriate here, as this is not what is normally understood as a wedge product.

> [wedge](https://en.wikipedia.org/wiki/Wedge_sum) can be misleading here

I actually agree, but a counterpoint is that that use of 'wedge' here demands the learner to learn something more about the wedge product, by drawing their attention to a space for comparison. Naming it so begs the question of why the shrinking wedge of circles is not a wedge sum.

Anyways, I felt the need to counter-argue just because I'm the one who suggested the alias. I did not know that there is a controversy surrounding the name when I brought up the Hawaiian earring space. I have no special interest in this, though after reading the various arguments I've come to believe that the 'Hawaiian' variant is unfortunate and misleading.

Reply:
Actually, I still think the name Hawaiian earring would be preferable as the primary name, as it is the least ambiguous and the canonical name used in the literature.  Opinions may differ.

Reply:
Regarding the proposed alias here, we could decide to not add it as an alias, but instead add a paragraph in the text.  Maybe something like "Appears as Example 1.25 in {{Hatcher}} under the name "Shrinking wedge of circles", which is a misleading name as it is not a wedge sum of circles".

Reply:
I'd lean towards adding the "wedge" alias since it is something a user might see in the textbook and attempt to look up in pi-Base.

I agree with @prabau that we should clarify in our description that is it not a wedge sum/product. I might suggest:

> Appears as Example 1.25 in {{Hatcher}} under the name "Shrinking wedge of circles"; however, note that it is not actually an example of a "wedge sum" as defined in e.g. {{wikipedia:Wedge_sum}}, {{doi:10.1007/978-1-4612-4576-6}}.